### ISSUE TITLE
Failed resolutions when serving an ESM package

### ISSUE THREAD

**Issue by wyrmisis** [break-stuff/cem-plugin-react-wrapper#2]
Hi there! I think I'm having an issue with this tool and my strict ES Module library.

When this tool creates React-specific components, it creates an index file composed of a number of `export * from './SomeComponent';`. Additionally, inside the generated components, `./react-utils`  is imported.

When attempting to consume a package of `"type": "module"` that uses these exported components, I get an error in the consuming application:

```
ERROR in `packagename/react/index.js`
Module not found: Error: Can't resolve "./Component" in "path/to/package"
Did you mean `Component.js`?
BREAKING CHANGE: The request './Component' failed to resolve only because it was resolved as fully specified (probably because the origin is strict EcmaScript module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type: "module"'). The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
``` 

If I manually add a `.js` extension to the generated `react/index.js` file, the same error comes up for `./react-utils`.

Maybe this tool ought to add filetypes to imports if the related `package.json` has `"type": "module`, or something similar?

**Comment by break-stuff** [comment]
Thanks for the heads-up! I think that should be pretty easy to resolve.

**Comment by break-stuff** [comment]
This latest patch release should fix the issue (2.1.2). Thank you!
